lib/ruby_lsp/document.rb

```diff
@@ -209,7 +209,7 @@ def find_char_position(position)
         # need to adjust for surrogate pairs
         requested_position = @pos + position[:character]
 
-        if @encoding == Constant::PositionEncodingKind::UTF16
+        if @encoding == Encoding::UTF_16LE
```

Comment:
Why `Encoding::UTF_16LE` rather than `Encoding::UTF_16`?

Reply:
I honestly can't remember the exact reason, but when Kevin and I were implementing the code units API we realized that using the generic UTF_16 did not work properly. It needed to specify `LE` or `BE` and since VS Code uses `UTF_16LE`, we use that.